### DISCUSSION TITLE
Who uses ghz?

### DISCUSSION THREAD

**Discussion by bojand** [bojand/ghz#38] [General]
Hey everyone, I'm curious to hear which companies or projects are using `ghz` and how? It would be inspiring and interesting to hear how are people finding this project useful.

 If you would like to help ghz - any of the following is greatly appreciated.

- [ ] Give the repository a star ⭐️
- [ ] Help out with issues
- [ ] Improve documentation
- [ ] Give [support ](https://www.buymeacoffee.com/bojand)

Thank you! 💐

**Comment by david-garay** [comment]
Hi There, 

Here in The Netherlands we are using for a (small) Master Engineering research project. I intend to use ghz for our experimentations, and find it a neat tool - thank you for it!

David

**Comment by jayhuang75** [comment]
Me here. From Canada.

**Comment by kenju** [comment]
Hi @bojand, I am kenju from Tokyo, Japan. I am a software engineer & a tech lead of [Cookpad](https://www.cookpadteam.com).

We have used ghz last year ([slide](https://speakerdeck.com/kenju/the-guide-to-load-test-grpc-application)) and it was really great helpful. We needed a gRPC load testing tool against a TV-CM burst RPS for live video applications.

Now, we are considering to use ghz again in a new project, advertisement servers. 

Though I am the only person who is using ghz in our company, but I hope I can help to develop & add more documentation & use-cases for ghz.

**Comment by arinto** [comment]
Hi @bojand, I'm Arinto from Singapore, working as software engineer in [Traveloka](https://www.traveloka.com/en/careers). Currently we're using ghz as our load testing tool for gRPC services in my team. We spawn several ghz processes using Kubernetes job in our staging cluster to emulate the clients of our gRPC services. 

Really appreciate ghz as it's pretty neat and do the job well :)

**Comment by craigh1015** [comment]
Hi @bojand - Craig from SanFransisco. I am evaluating using ghz for load testing of a set of new gRPC microservices we are building.

**Comment by gearnode** [comment]
At @jobteaser we use it, thanks for the tool ❤️

**Comment by gurre** [comment]
I used ghz to run a load test on AWS DynamoDB pushing 500k requests per second. It used 290 c5n.xlarge EC2 instances on ECS running 200 ghz containers against 200 consumers.

**Comment by maximveksler** [comment]
Hello 👋

Here at [Seed-X](https://seed-x.com/) we've used ghz to get capacity estimates for our inference infrastructure. 

We're currently researching options for extending ghz into a locust.io like distributed benchmarking tool.

**Comment by Shobhit29** [comment]
I am using it to test our GRPC microservice. I find it a very neat tool. But I am trying to understand how we can write the report to a file with the Go Program instead of STDOUT. Can you give an example of it
